src/common/components/feedback-form/FeedbackForm.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import ChipSelection from '../chip-selection/ChipSelection';
+import ContactInputField from '../contact-input-field/ContactInputField';
+import Textarea from '../textarea/Textarea';
+import { FeedbackFormConfig } from './configs/FeedbackForm.config';
+import confirmLogo from '../../../assets/icons/confirm-icon.svg';
+
+const FeedbackForm = () => {
+  const [showForm, setShowForm] = useState<boolean>(true);
```

Comment:
Nu-i mai bine sa fie invers? 
Sa ai o variabila "showSuccess" sau "showSuccessCard" sau ceva de genul?

Reply:
Trebuie inversata logica de afisare, daca esti de acord cu ce am zis mai sus

Reply:
Modificat in 'showSuccess'.